src/time.cpp

```diff
@@ -69,6 +71,28 @@ namespace g3 {
    /// * format string must conform to std::put_time
    /// This is similar to std::put_time(std::localtime(std::time_t*), time_format.c_str());
 
+   namespace {
+      std::string kNanoSecondIdentifier = "%f";
```

Comment:
OK. I buy the ```%f``` since it will relate to many people already familiar with boost
http://www.boost.org/doc/libs/1_35_0/doc/html/date_time/date_time_io.html